src/AppInstallerCLICore/Commands/PinCommand.cpp

```diff
@@ -119,14 +117,12 @@ namespace AppInstaller::CLI
             // When we are given an ID, just pin that available package without checking for installed.
             // This helps when there are matching issues, for example due to multiple side-by-side installs.
             context <<
-                Workflow::OpenSource();
+                Workflow::OpenSourceFor(OperationType::Search);

```

Comment:
Also not a fan of having to use ::Search in the Pin command

Reply:
Understandable; Will adjust